DirectProgramming/DPC++/CombinationalLogic/sepia-filter/README.md

```diff
@@ -52,7 +52,7 @@ Perform the following steps:
 ### On a Windows* System Using Visual Studio* version 2017 or Newer
 
 * Build the program using VS2017 or VS2019: Right click on the solution file and open using either VS2017 or VS2019 IDE. Right click on the project in Solution explorer and select Rebuild. From top menu select Debug -> Start without Debugging.
-* Build the program using MSBuild: Open "x64 Native Tools Command Prompt for VS2017" or "x64 Native Tools Command Prompt for VS2019". Run - MSBuild mandelbrot.sln /t:Rebuild /p:Configuration="Release"
+* Build the program using MSBuild: Open "x64 Native Tools Command Prompt for VS2017" or "x64 Native Tools Command Prompt for VS2019". Run - MSBuild sepia-filter.sln /t:Rebuild /p:Configuration="Release"
 
 
 ## Running the sample
```

Comment:
Readme needs to have a statement about using include folder.

Reply:
Readme does not comply with current standards, Need to update to reflect this standard format  https://gitlab.devtools.intel.com/ecosystem-dev-programs/pre-release/intel-oneapi-sample-template/blob/master/README.md

Reply:
@anjgola Line 27 of Readme includes the line about include folder, anything else that I am missing?

Reply:
@JoeOster Updated Readme, though I did not notice anything major that changed in the template since last week. Please let me know if there is anything specific that is missing.

Reply:
Basically, you need the title, description and purpose sections added to the readme

Reply:
@anjgola @JoeOster Are you'll seeing the latest Readme that is at https://github.com/sravanikonda/oneAPI-samples/blob/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/sepia-filter/README.md? All the changes requested above are already in there.

Reply:
I was looking at the file changed view and it didn't show, but when I looked in your repo it is indeed correct.